rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -529,14 +530,16 @@ def test_maybe_detect_new_spam_tokens(
 
 @pytest.mark.vcr(filter_query_parameters=['apikey'])
 @pytest.mark.parametrize('gnosis_accounts', [['0xcC3Da35614E6CAEEA7947d7Cd58000C350E7FF84']])
+@pytest.mark.parametrize('ethereum_accounts', [['0xb524c787669185E11d01C645D1910631e04Fa5Eb']])
 def test_maybe_augmented_detect_new_spam_tokens(
         task_manager: TaskManager,
         database: 'DBHandler',
         globaldb: GlobalDBHandler,
         gnosis_inquirer,
+        ethereum_inquirer,
 ) -> None:
     """
-    Test the agumented spam detection schedule and behaviour. We use a token that is not detected
+    Test the augmented spam detection schedule and behavior. We use a token that is not detected
```

Comment:
Behaviour -> British english. 
Behavior -> American english

```suggestion
    Test the augmented spam detection schedule and behaviour. We use a token that is not detected
```